do_droplet_import_test.py

```diff
@@ -0,0 +1,111 @@
+
+        try:
+            list_of_providers_field.find_element_by_link_text(settings.provider_name)
+        except NoSuchElementException:
+            print 'Created provider is found. This is good.'
+        else:
+            print 'I cannot find created provider, test fails.'
+        
+        print 'Provider was created successfully.'
+        time.sleep(10)
+        print 'Resource is going to be imported.'
+        import_resource(self.driver, settings.project_name, settings.provider_name, settings.resource_name)
+        time.sleep(5)
+        list_of_resource_field = self.driver.find_element_by_css_selector('[ng-model="generalSearch"]')
+
+        try:
+            list_of_resource_field.find_element_by_link_text(settings.resource_name)
+        except NoSuchElementException:
+            print 'Imported resource is found. This is good.'
+        else:
+            print 'I cannot find imported resource, test fails.'
+        print 'Resource was imported successfully.'
+        time.sleep(10)
+        print 'Resource is going to be unlinked.'
+        unlink_resource(self.driver, settings.project_name, settings.resource_name)
+        time.sleep(10)
+        list_of_resource_field = self.driver.find_element_by_css_selector('[ng-model="generalSearch"]')
+
+        try:
+            list_of_resource_field.find_element_by_link_text(settings.resource_name)
+        except NoSuchElementException:
+            print 'I cannot find unlinked resource. This is good.'
+        else:
+            print 'Unlinked resource is found, test fails.'
+        print 'Resource was unlinked successfully.'
+        print 'Provider is going to be removed.'
+        delete_provider(self.driver, settings.project_name, settings.provider_name)
+        time.sleep(10)
+        list_of_provider_field = self.driver.find_element_by_css_selector('[ng-model="generalSearch"]')
+
+        try:
+            list_of_provider_field.find_element_by_link_text(settings.provider_name)
+        except NoSuchElementException:
+            print 'I cannot find removed provider. This is good.'
+        else:
+            print 'Removed provider is found, test fails.'
+        print 'Provider was removed successfully.'
+
+    def tearDown(self):
+        if self.is_project_created:
+            try:
+                delete_project(self.driver, settings.project_name)
+                # Check deletion
+                time.sleep(10)
+                list_of_projects_field = self.driver.find_element_by_class_name('object-list')
+
+                try:
+                    list_of_projects_field.find_element_by_link_text(settings.project_name)
+                except NoSuchElementException:
+                    print 'I cannot find project. This is good.'
+                else:
+                    print 'Project is found, test fails.'
+            except Exception as e:
+                print 'Project cannot be deleted. Error: %s' % e
```

Comment:
Provider has to be removed in `tearDown` method too. (If link operation fails - we need to remove provider anyway.)